src/components/domain/user/CirclePlanetIcon.tsx

```diff
@@ -10,42 +9,23 @@ interface CirclePlanetIconProps {
   iconIndex?: number;
 }
 
-const CirclePlanetIcon = ({ className, iconIndex }: CirclePlanetIconProps) => {
+export default function CirclePlanetIcon({ className, iconIndex }: CirclePlanetIconProps) {
```

Comment:
```suggestion
export default function CirclePlanetIcon({ className, iconIndex = 0 }: CirclePlanetIconProps) {
```
여기서 0으로 지정하고 아래에 삼항연산자를 간소화하는게 깔끔할 것 같습니다

Reply:
✅ [627d4e9](https://github.com/PRism-Works/PRism-FE/pull/96/commits/627d4e967ea797eed204da08ff078a9770bd4570)